specification.md

```diff
@@ -0,0 +1,152 @@
+      action: denied
+      path:
+        - main/foo/bar
+        - main/examples
+      comment: |
+        Lorem ipsum dolor sit amet, consectetur adipisci elit,
+        sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.
+  contributing-policy: https://example.com/development-policy.html
+  code-of-conduct: https://example.com/code-of-conduct.html
+```
+
+- `accepts-pull-requests` (Required): Does your project accept PRs from non-maintainers?
+  - **Boolean**
+- `accepts-automated-pull-requests` (Required): Does your project accept PRs from bots or automated tools?
+  - **Boolean**
+- `automated-tools-list`: If you have exceptions to accepts-automated-pull-requests, such as allowing select tools or blocking select tools, review the specification for more information about how to list those exceptions here.
+  - **List** of **Objects**
+- `contributing-policy`: URI to your project’s detailed contribution policy.
+  - **String** beginning with **https://**
+- `code-of-conduct`: URI to your project’s code of conduct.
+  - **String** beginning with **https://**
+
+## Documentation
+The documentation section is simply a list of URIs to any security documentation you have provided for stakeholders.
+
+```yaml
+documentation:
+  - http://foo.bar/wiki
+```
+
+This section is not required. Each entry to the list must be a **String** beginning with **https://**.
+
+## Distribution Points
+Similar to the previous section in our YAML, the distribution-points section is simply a list of URIs. This list should include any locations that users can expect to safely retrieve your released software.
+
+```yaml
+distribution-points:
+  - https://example.com/foo
+  - pkg:npm/foobar
+```
+
+This section is not required. Any entry to the list must be a **String**.
+
+## Security Artifacts
+The security-artifacts section allows you to share any additional security documentation that lives outside this document.
+
+```yaml
+security-artifacts:
+  threat-model:
+    threat-model-created: true
+    evidence-url:
+    - https://foo.com/model.html
+    comment: |
+      Lorem ipsum dolor sit amet, consectetur adipisci elit,
+      sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.
+```
+
+This section is not required. If threat-model-created is true, then evidence-url must contain at least one **String** beginning with **https://**.
+
+## Security Testing
+The security-testing section contains a list of tools your project has incorporated for automated security testing. This section is not required, but there are required and optional values for each entry, which we’ll look at below.
+
+- `tool-name` (Required): **String**
+- `tool-type` (Required): **String**, values may be sast, dast, iast, fuzzer
+- `tool-version` (Required): **String**
+- `tool-url`: **String** beginning with **https://**
+- `integration` (Required)
+    - `ad-hoc` (Required): Is the tool used manually on an ad-hoc basis?
+    - **Boolean**
```

Comment:
I think the formatting here is off, above the data type is indented more.

Reply:
updated